src/LGraphNode.ts

```diff
@@ -2941,6 +2949,70 @@ export class LGraphNode implements Positionable, IPinnable {
     }
   }
 
+  /**
+   * Renders the node's title text.
+   */
+  drawTitleText(ctx: CanvasRenderingContext2D, options: {
+    scale: number
+    default_title_color?: string
+    low_quality?: boolean
+    title_height?: number
+  }): void {
+    const {
+      scale,
+      default_title_color,
+      low_quality = false,
+      title_height = LiteGraph.NODE_TITLE_HEIGHT,
+    } = options
+
+    const size = this.renderingSize
+    const selected = this.selected
+
+    if (this.onDrawTitleText) {
+      this.onDrawTitleText(
+        ctx,
+        title_height,
+        size,
+        scale,
+        this.titleFontStyle,
+        selected,
+      )
+      return
```

Comment:
Behavior change. `onDrawTitleText` now override the text rendering to match the other callbacks. However currently this callback is unused, so this behavior change won't affect any existing code.